### PR TITLE
[receiver/dockerstats] remove duplicate Timeout setting

### PR DESCRIPTION
**Pull request by codeboten**
Use the Timeout setting from ScraperHelper instead

### CODE REVIEW

**Review by crobert-1 — APPROVED** [review]

**Comment by crobert-1** [comment]
I filed #26123 to address the CI failure, it's unrelated to this change.

**Review by atoulme — APPROVED** [review]

File: .chloggen/codeboten_rm-dupe-dockerstats.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```
Comment (jamesmoessis) [inline .chloggen/codeboten_rm-dupe-dockerstats.yaml:4]: Is this really a breaking change if there is no impact to users? Seems to be an enhancement to me.

**Review by jamesmoessis — APPROVED** [review]
Nice, makes it easy that the scraperhelper timeout has the same signature.

File: .chloggen/codeboten_rm-dupe-dockerstats.yaml (hunk 1 above)
Comment (codeboten) [reply .chloggen/codeboten_rm-dupe-dockerstats.yaml:4]: It's a breaking change for the API if anyone was instantiating the configuration for the component programmatically

**Pull request merged**